### PR TITLE
Prac 07 Feedback

### PR DESCRIPTION
**Pull request by dylanryder708**
Hi @KoenMeecham could you please review my code.

### CODE REVIEW

File: prac_07/guitar.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        """Initialise a Guitar."""
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Return a string representation of a Guitar."""
+        return f"{self.name} ({self.year}) : ${self.cost:,.2f}"
+
+    def __lt__(self, other):
```
Comment (KoenMeecham) [inline prac_07/guitar.py:23]: Put a docstring under here, maybe "Return if self is > other"

File: prac_07/myguitars.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
+import csv
```
Comment (KoenMeecham) [inline prac_07/myguitars.py:1]: Add a docstring at the top and under the main function,  but the comments are really good

File: prac_07/project.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Finish Time: 7:55 PM
+"""
+import datetime
+
+
+class Project:
+    """A class to represent a project."""
+
+    def __init__(self, name, start_date, priority, estimate, completion):
+        """Initialize a project with name, start date, priority, cost estimate, and completion percentage."""
```
Comment (KoenMeecham) [inline prac_07/project.py:13]: The class shouldn't have to be input checking, this should be done in the main code (try except things).

File: prac_07/project_management.py [hunk 4]
```diff
@@ -0,0 +1,123 @@
+import csv
```
Comment (KoenMeecham) [inline prac_07/project_management.py:1]: Add a docstring here, i personally just copy and paste the docstring in the class function

File: prac_07/project_management.py [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+            update_project(projects)
+        else:
+            print("Invalid option")
+    save_choice = input(f"Would you like to save to {filename}? (y/n): ").lower()
+    if save_choice == 'y':
+        save_projects(filename, projects)
+    print("Thank you for using custom-built project management software.")
+
+
+def load_projects(filename):
```
Comment (KoenMeecham) [inline prac_07/project_management.py:43]: You can make filename a constant and not worry about having to pass it into functions

**Comment by dylanryder708** [comment]
Thank you for the feedback!

**Pull request merged**